src/com/jedk1/jedcore/JCMethods.java

```diff
@@ -17,27 +23,34 @@
 import org.bukkit.inventory.ItemStack;
 import org.bukkit.util.Vector;
 
-import com.jedk1.jedcore.configuration.JedCoreConfig;
-import com.jedk1.jedcore.scoreboard.BendingBoard;
-import com.projectkorra.projectkorra.GeneralMethods;
-import com.projectkorra.projectkorra.ProjectKorra;
-import com.projectkorra.projectkorra.ability.CoreAbility;
-import com.projectkorra.projectkorra.ability.util.ComboManager;
-import com.projectkorra.projectkorra.util.TempBlock;
+import java.util.*;
```

Comment:
this doesn't matter much but next time can you not rearrange the imports

Reply:
Yes, I just didn't knew I had default setting in IntelliJ IDEA at that moment. Now I set that imports will never be changed like that.